requirements.txt

```diff
@@ -4,6 +4,7 @@ numpy
 pytest
 pyvisfile
 pymetis
+importlib-resources>=1.1.0; python_version < '3.9'
```

Comment:
This change probably broke emirge.

Reply:
Oh hrm, I haven't had any troubles with it, but maybe I would need --branch-name=<something> to catch it?

Reply:
> Oh hrm, I haven't had any troubles with it, but maybe I would need --branch-name= to catch it? 

Any fresh installation of emirge will fail, see e.g. here https://github.com/illinois-ceesd/emirge/runs/1865330076

> (edit: pip thinks it's ok).

Emirge doesn't use pip to parse this file.

What is the intention here, why not install this package for python 3.9 too?

Reply:
For python >= 3.9, importlib-resources.files is baked-in.  See e.g.
https://jwodder.github.io/kbits/posts/pypkg-data/#accessing-package-data-at-runtime

Reply:
Sure, but installing the package won't hurt, right?

Reply:
🤷 Elephino